styles.css

```diff
@@ -50,7 +50,7 @@ i {
   filter: grayscale(100%);
   list-style-type: none;
   margin-right: 5em;
- 
+  -webkit-filter: sepia(100%) hue-rotate(200deg); 
```

Comment:
Just as the previous reviewer suggested, Kindly ensure your icon color matches the given design😉 You still haven't made the changes as regards that🥲.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="300" alt="yours" src="https://user-images.githubusercontent.com/64643995/234832532-7925b4e4-ed3c-4ff1-a9c7-437b49e73c08.png">|<img width="300" alt="design" src="https://user-images.githubusercontent.com/64643995/234832612-896e229c-5201-41af-b3c7-75ac145cfd2d.png">|